interface/weno/import_pharmacies.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
Use a processing gif that does not contain word(s) in it, since this is impossible to translate.